### DISCUSSION TITLE
zh/datasets/segment/package-seg/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#19044] [Docs]
# zh/datasets/segment/package-seg/

探索Roboflow 包裹分类数据集。利用用于包装识别和分拣的策划图像优化物流并增强视觉模型。

https://docs.ultralytics.com/zh/datasets/segment/package-seg/

**Comment by UltralyticsAssistant** [comment]
👋 Hello, thank you for bringing this up and for your interest in Ultralytics 🚀! It looks like you're exploring the Roboflow package segmentation dataset. That's fantastic! We recommend referring to the [Docs](https://docs.ultralytics.com/zh/datasets/segment/package-seg/) for dataset-specific details and guidance on how to utilize such datasets effectively in your projects.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) so we can assist with debugging. This will allow us to understand and address the issue more effectively.

For any custom training ❓ Questions, kindly include as much detail as possible, like dataset examples, training logs, hyperparameters, and any specific challenges you're encountering. Make sure you're also following our best practices outlined in the [Model Training Tips](https://docs.ultralytics.com/guides/model-training-tips/) to optimize your performance.

Here are some additional resources and steps you may find helpful:

## Upgrade

Ensure your `ultralytics` package is up to date along with all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/). Update your packages using the command below to ensure you're running the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO can be run in these environments—each preconfigured with dependencies for seamless setup:

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM: Refer to the [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI: Refer to the [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**: Refer to the [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Community and Support

💬 Join the Ultralytics community that suits you best!  
- For interactive discussions, jump into our [Discord](https://discord.com/invite/ultralytics) 🎧.
- For structured, long-form discussions, participate on [Discourse](https://community.ultralytics.com/).
- Or connect with others and share your thoughts on our [Subreddit](https://reddit.com/r/Ultralytics).

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>  
The current status of [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) provides real-time insights into our integration tests, ensuring compatibility across various platforms and tasks.

This is an automated response 😊, but no worries—an Ultralytics engineer will follow up soon to assist you further!
